packages/volto/src/helpers/Loadable/__mocks__/Loadable.jsx

```diff
@@ -18,21 +18,21 @@ export const __setLoadables = async () => {
 };
 
 // TODO: filter mockAllLoadables
-export const injectLazyLibs = vi.fn().mockImplementation(function ([
+export const injectLazyLibs = jest.fn().mockImplementation(function ([
```

Comment:
At the beginning of all Jest mocks, I would add the following comment:


```javascript
/* FIXME: When Jest configuration support is removed, update this file with the vitest version of the mock. Then, in the tests, we can replace:

vi.mock('@plone/volto/helpers/Loadable/Loadable', async () => {
  return await import(
    '@plone/volto/helpers/Loadable/__mocks__/Loadable.vitest.jsx'
  );
});

With:

vi.mock('@plone/volto/helpers/Loadable/Loadable');
*/
```

Reply:
@wesleybl I'm not clear to whom this comment is addressed. Is it Volto core contributors, or is it project developers, or both? It matters because of the use of the passive voice used in the phrase, "When Jest configuration support is removed", specifically who will do the removal?

Reply:
OK, then let's change that phrase to make it explicit.

```
When you remove Jest configuration support from your add-on,
```

Reply:
@stevepiercy No. This removal must be done in Volto. I think it was agreed that in Voto 19 the Jest configuration will be removed.